Source/APIClient/Request.swift

```diff
@@ -19,6 +19,7 @@ public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
     static var authToken: String? { get }
+    static var authTokenHeader: String? { get }
```

Comment:
@markdintrepid Might we ever want access to _just_ the `token`, though, without the rest of the header value?

For example, with an API I'm working with now, there is an example of a request where the `authToken` is passed as a body parameter value, in which case we would want to isolate the token itself from the `Token token=`.